src/profiles.jl

```diff
@@ -36,49 +36,76 @@ Inputs:
 - `costs::Vector{Function}`: a vector of functions specifying the measures to use in the profiles
 - `costnames::Vector{String}`: names to be used as titles of the profiles.
 
+Keyword inputs:
+- `width::Int`: Width of each individual plot (Default: 400)
+- `height::Int`: Height of each individual plot (Default: 400)
+
 Output:
 A Plots.jl plot representing a set of performance profiles comparing the solvers.
 The set contains performance profiles comparing all the solvers together on the
 measures given in `costs`.
 If there are more than two solvers, additional profiles are produced comparing the
 solvers two by two on each cost measure.
 """
-function profile_solvers(stats::Dict{Symbol,DataFrame}, costs::Vector{<:Function}, costnames::Vector{String})
+function profile_solvers(stats::Dict{Symbol,DataFrame},
+                         costs::Vector{<:Function},
+                         costnames::Vector{String};
+                         width::Int=400,
+                         height::Int=400
+                        )
   solvers = collect(keys(stats))
   dfs = (stats[solver] for solver in solvers)
   Ps = [hcat([cost(df) for df in dfs]...) for cost in costs]
 
   nprobs = size(stats[first(solvers)], 1)
+  nsolvers = length(solvers)
+  ncosts = length(costs)
+  npairs = div(nsolvers * (nsolvers - 1), 2)
 
   # profiles with all solvers
   ps = [performance_profile(Ps[1], string.(solvers), title=costnames[1], legend=:bottomright)]
-  for k = 2 : length(Ps)
-    push!(ps, performance_profile(Ps[k], string.(solvers), title=costnames[k], legend=false))
+  if nsolvers > 2
+    xlabel!(ps[1], "")
+  end
```

Comment:
Could we make this a one liner: `nsolver > 2 && xlabel!(ps[1], "")`?

Reply:
Done